docs/OCP_CI_Tutorials/Scenarios/Reporting_Guide.md

```diff
@@ -0,0 +1,128 @@
+
+### How to Create a New Dashboard
+
+Creating a new TestGrid dashboard is very simple. Before you get started you will need to [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) the [kubernetes/test-infra](https://github.com/kubernetes/test-infra) repository into your personal GitHub organization.
+
+
+Once you have your fork created, follow these instructions to create a new dashboard:
+
+1. Prepare Git:
+   1.  Clone a copy of your fork to your computer
+   2.  Create a new branch and name it whatever you'd like.
+2. In your new branch, create a new file in the `config/testgrids/openshift` directory. The name of the file should be the same name as your new dashboard
+   - **Example:** `config/testgrids/openshift/redhat-openshift-4.12-interop.yaml`
+
+> **IMPORTANT**:
+>
+> Do not use `-release-` at all in your file name or your dashboard name. This will break the OpenShift CI automation that generates their TestGrid dashboards. See this pull request for more information: [kubernetes/test-infra PR #28593](https://github.com/kubernetes/test-infra/pull/28593 )
+
+3. Write your configuration file.
+   - **Note:** Use the example in the [TestGrid Configuration Example and Description](#testgrid-configuration-example-and-description) section of this document.
+4. After you have pushed your changes to your fork, follow the [pull request process](#pull-request-process) to submit a PR and have your dashboard created.
+
+### How to Add a Job to a Dashboard
+
+If there is already a suitable dashboard created in TestGrid to hold your new Prow job, follow this guide to get it appended to the existing dashboard. Before you get started you will need to [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) the [kubernetes/test-infra](https://github.com/kubernetes/test-infra) repository into your personal GitHub organization.
+
+1. Prepare Git:
+   1.  Clone a copy of your fork to your computer
+   2.  Create a new branch and name it whatever you'd like.
+2.  In your new branch, find the configuration file for the dashboard you'd like to use.
+    - **Note**: The config files should be in the `config/testgrids/openshift` directory and the name of the file should be the same as the name on the TestGrid website.
+    - **Example:** `config/testgrids/openshift/redhat-openshift-4.12-interop.yaml`
+3. Create an entry in the `test_groups` stanza of the config file.
+    - **Note:** Use the example in the [TestGrid Configuration Example and Description](#testgrid-configuration-example-and-description) section of this document.
+    - **Note:** The `name` value in this item needs to be unique, not just in the document but in the other configs as well. You may want to search the repo for the string you decide on to make sure it isn't used.
```

Comment:
```suggestion
    - **Note:** The `name` value in this item needs to be unique, not just in the document, but in the other configs as well. You may want to search the repo for the string you decide on to make sure it isn't used.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)